blastradius/handlers/terraform.py

```diff
@@ -53,6 +53,9 @@ def __init__(self, directory=None, settings=None):
                     continue
                 elif re.match(r'git\:\:.*', source):
                     continue
+                # Public Terraform Module Registry
+                elif re.match(r'^[a-zA-Z0-9]+\/[a-zA-Z0-9\-]+\/[a-zA-Z0-9]+', source):
```

Comment:
Are underscores allowed?

Reply:
@joestump [the docs](https://www.terraform.io/docs/registry/modules/publish.html) are unclear as to exactly what's allowed :disappointed:

Reply:
@joestump ok, can confirm, underscores are allowed: https://registry.terraform.io/modules/davewongillies/test-module/test_repo/1.0.0

I'll push another change to allow for that